src/lib/utils.js

```diff
@@ -488,6 +490,34 @@ const validateActorName = (actorName) => {
     }
 };
 
+const getPythonCommand = (directory) => {
+    const pythonVenvPath = /^win/.test(process.platform)
+        ? './.venv/bin/python3.exe'
+        : './.venv/bin/python3';
+
+    if (process.env.VIRTUAL_ENV) {
+        directory = process.env.VIRTUAL_ENV;
+    }
+    const fullPythonVenvPath = path.join(directory, pythonVenvPath);
```

Comment:
this won't work if you're in the VENV - `VIRTUAL_ENV` is `path/to/.venv`, so combined with `./.venv/bin/python` you have the `.venv` there twice

Reply:
Good catch, will fix 🙂